_includes/pbc/all/install-features/202212.0/install-the-agent-assist-feature.md

```diff
@@ -0,0 +1,528 @@
+
+Run the following command to install the required modules:
+
+```bash
+composer require spryker-feature/agent-assist:"{{site.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE      | EXPECTED DIRECTORY               |
+|-------------|----------------------------------|
+| AgentPage   | vendor/spryker-shop/agent-page   |
+| AgentWidget | vendor/spryker-shop/agent-widget |
+
+{% endinfo_block %}
+
+### 2) Set up configuration
+
+By default, in Spryker, posting login form (where SecurityBlocker will make its check and block agents who made too many failed login attempts) is locale-independent. So, to be able to see error messages translated into different languages, you need to configure the locale to be added to the agent login path. You can do this by modifying the following configs:
+
+**src/Pyz/Yves/AgentPage/AgentPageConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\AgentPage;
+
+use SprykerShop\Yves\AgentPage\AgentPageConfig as SprykerAgentPageConfig;
+
+class AgentPageConfig extends SprykerAgentPageConfig
+{
+    /**
+     * @return bool
+     */
+    public function isLocaleInLoginCheckPath(): bool
+    {
+        return true;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that when the login form for the agent is submitted, the URL it uses contains a locale code. For example, `/de/agent/login_check` would be the default value for the agent.
+
+{% endinfo_block %}
+
+### 3) Add translations
+
+Add translations as follows:
+
+1. Append the glossary according to your configuration:
+
+**src/data/import/glossary.csv**
+
+```yaml
+agent.authentication.failed,Authentication failed,en_US
+agent.authentication.failed,Authentifizierung fehlgeschlagen,de_DE
+agent.login.title,Access your account,en_US
+agent.login.title,Ich bin bereits Kunde,de_DE
+agent.confirm_user_selection,Confirm,en_US
+agent.confirm_user_selection,Bestätigen,de_DE
+agent.login.email,Email,en_US
+agent.login.email,E-Mail,de_DE
+agent.login.password,Password,en_US
+agent.login.password,Passwort,de_DE
+agent.control_bar.username,Agent: %username%,en_US
+agent.control_bar.username,Agent: %username%,de_DE
+agent.control_bar.customer_name,Customer: %username%,en_US
+agent.control_bar.customer_name,Kunde: %username%,de_DE
+agent.control_bar.logout_as_customer,End Customer Assistance,en_US
+agent.control_bar.logout_as_customer,Kunden-Assistenz beenden,de_DE
+agent.control_bar.logout,Logout,en_US
+agent.control_bar.logout,Abmelden,de_DE
+agent.autocomplete.no_results,No results found,en_US
+agent.autocomplete.no_results,Keine Ergebnisse gefunden,de_DE
+autocomplete.placeholder,Search,en_US
+autocomplete.placeholder,Suche,de_DE
+```
+
+2. Run the following command to add the glossary keys:
+
+```bash
+console data:import:glossary
+```
+
+### 4) Enable controllers
+
+Enable the following controllers.
+
+#### Service provider list
```

Comment:
This part has been deprecated for 3 years. We should replace this with `AgentPageSecurityPlugin` installation instructions.